### ISSUE TITLE
A-B repeat feature

### ISSUE THREAD

**Issue by Oleksiy-Yakovenko** [DeaDBeeF-Player/deadbeef#909]
Original [issue 1007](https://code.google.com/p/ddb/issues/detail?id=1007) created by Alexey-Yakovenko on 2013-11-27T11:08:21.000Z:

Looks like there is no &quot;A-B repeat&quot; feature implemented yet.

It would be fine to either advice how to make the feature available via plugins or implement in-place.

Many thanks!

**Comment by webmastak** [comment]
Why the functionality is not implemented, 5 years have passed!!!

**Comment by Alexey-Yakovenko** [comment]
Just like all the other features that are not implemented, it's usually because nobody was interested in doing the work.

**Comment by webmastak** [comment]
It would be nice if you implemented it.

**Comment by Alexey-Yakovenko** [comment]
Yeah, it would be. But there are other ~300 feature requests, and I'm working on things 1 at a time.
Obviously 1 lifetime is not enough to get to everything.
